include/ConfigManager.h

```diff
@@ -45,7 +45,7 @@ const QString TEMPLATE_PATH = "templates/";
 const QString PRESETS_PATH = "presets/";
 const QString SAMPLES_PATH = "samples/";
 const QString GIG_PATH = "samples/gig/";
-const QString SF2_PATH = "samples/soundfonts/";
+const QString SoundFont_PATH = "samples/soundfonts/";
```

Comment:
(Non-blocking) Per the [Coding Conventions](https://github.com/LMMS/lmms/wiki/Coding-Conventions#global-constants):

> Global scope constants SHALL use `UpperCamelCase`.

```suggestion
const QString SoundFontPath = "samples/soundfonts/";
```

It may look out of place next to all the other `UPPER_SNAKE_CASE` constants; you can update those too if you feel like doing so.